### DISCUSSION TITLE
join-method for arrays wont show space

### DISCUSSION THREAD

**Discussion by AntjeWalter** [neuefische/questions#391] [Web] [answered]
Hi everyone, 

I am working on a table where I want to include several ideas in the same cell. The ideas are stored in an array of objects like this:
`{
    id: 1,
    name: "Max Mustermann",
    birthday: "01.01.2001",
    idea: ["idea1", "idea2", "idea3", "idea4", "idea5"],
  },`

I wanted to display all the ideas in a single cell and used the join-method like this:

`<td>{[birthday.idea].join(", ")}</td>`

However, the comma works and all ideas are shown but the space doesn't show. 
mdn says that using ", " should work and show all strings divided with a comma and a space. 

[Here](https://capstone-project-git-feature-list-with-ideas-antjewalter.vercel.app/) is a preview of my app and here is my [PR](https://github.com/AntjeWalter/capstone-project/pull/3/files).

**Comment by MerleMak** [comment, marked as answer]
Hi Antje, 
One problem might be your dummy data. In the first object you are using an array for the idea(s*!) key. In the other objects, idea is a string. 
I would try to build up you dataset, so it has the same datatype for each key. 

Secondly, you don't need the `[ ]` around `birthday.idea`, as the array is already declared inside of your data set :)
So
`<td>{birthday.idea.join(", ")}</td>` 
should work once your dataset is fixed :))

Good luck!
